sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -181,11 +184,30 @@ private Tree convertElementToSlangAST(PsiElement element, TreeMetaData metaData)
       return createOperationExpression(metaData, (KtOperationExpression) element);
     } else if (element instanceof KtParameter) {
       return createParameter((KtParameter) element);
+    } else if (element instanceof KtProperty) {
+      return createVariableDeclaration((KtProperty) element);
     } else {
       return convertElementToNative(element, metaData);
     }
   }
 
+  private Tree createVariableDeclaration(KtProperty ktProperty) {
+    TreeMetaData metaData = getTreeMetaData(ktProperty);
```

Comment:
you could directly pass the metada as parameter as it is computed in parent method, the way it is done for other `create` methods.